sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/CalciteQueryPlanner.java

```diff
@@ -75,7 +75,9 @@
   private final Planner planner;
   private final JdbcConnection connection;
 
-  public CalciteQueryPlanner(JdbcConnection connection, RuleSet[] ruleSets) {
+  public CalciteQueryPlanner(
+      JdbcConnection connection, RuleSet[] ruleSets, String defaultTimezone) {
+    // TODO: support setting default timezone?
```

Comment:
If defaultTimezone is set here, we should warn the user it will have no effect.

Reply:
See comment below.